### ISSUE TITLE
In-house forecasting team

### ISSUE THREAD

**Issue by OAGr** [QURIresearch/potential-projects#8]
Have 2-4 people, in-house, doing forecasting of key things. This would be used for forecasting with low weights/stars. It would hopefully speed up work by a lot, because right now we need to wait on Metaculus for most things (this requires waiting for them to post the questions, which could take months, then hoping/waiting for forecasters to forecast on them)

**Comment by uvafan** [comment]
I like this idea overall (it could even spin off into a competitor to Good Judgment superforecasters if it goes well). Why do you propose using it for forecasting with low weights/stars?

**Comment by OAGr** [comment]
They seem more neglected. There's a decent bar for metaculus questions.

**Comment by uvafan** [comment]
My independent impression pretty strongly disagrees with this or I'm misunderstanding you. Would be good to chat about this too

**Comment by OAGr** [comment]
My fairly strong impression is that lots and lots of interesting stuff isn't estimated. I think this has been a major oversight of academia. For example, think of "microcovid, but for many other things". This might be a longer discussion though.

**Comment by NunoSempere** [comment]
I think that Eli disagreed with Metaculus being a decent bar, not with it being more neglected.

**Comment by NunoSempere** [comment]
I agree this proposal would just straightforwardly be good. I think the main bottleneck is actually QURI funds here.
